.pullapprove.yml

```
@@ -1050,28 +1048,26 @@ groups:
   # =========================================================
   code-ownership:
     conditions:
+      - *can-be-global-approved
       - >
         contains_any_globs(files, [
           '.pullapprove.yml'
           ])
     reviewers:
       users:
         - IgorMinar
-      teams:
-        - ~framework-global-approvers
 
 
   # ====================================================
   #  Catch all for if no groups match the code change
   # ====================================================
   fallback:
     conditions:
+      - *can-be-global-approved
       # Groups which are found to have matching conditions are `active`
       # according to PullApprove. If no groups are matched and considered
       # active, we still want to have a review occur.
       - len(groups.active) == 0
     reviewers:
       users:
         - IgorMinar
```

I know that this is unrelated, but could we actually remove me from here (in a separate PR) and have the PR fail because no reviewer has been assigned to it and force us to update the rules to cover the missing paths?

currently some of these reviews come to me and I update the pullrequest, but other times I don't update it because I don't realize that I'm asked to review this because I'm the final fallback. I'd prefer to get myself out of this and let the PR crash and require pullapprove config change (with global approvers still being around for emergency situations)

---

Will do